crates/biome_css_formatter/tests/specs/scss/at-rule/each.scss

```
@@ -40,3 +40,27 @@ puma,sea-slug,egret,salamander{
 
 @each $animal in /* animals */ $animals{
 }
+
+@each $animal in((puma),(sea-slug),(egret),(salamander)){}
+
+@each $element,$size in(h1:20px,h2:16px,h3:14px){}
```

Stylelint (17.9.0)

[error] Empty block (block-no-empty)

(block-no-empty)